Automattic/newspack-plugin
# fix: partial text domain update for the newspack plugin (#2646)

### All Submissions:


### Changes proposed in this Pull Request:

This PR moves #2646 into a hotfix; steps to test are copied from there. 

### How to test the changes in this Pull Request:

1. Visually review the PR and make sure the only changes that have been made are changing newspack to newspack-plugin in the translation strings, and greatly reducing the POT file (only strings with the updated text-domain should show up there).
2. Test to make sure translations actually load -- for this, can make your own .PO file and translate some of the strings and make sure they show up on the front-end. I also have a fake PO and MO German translation you can use - you can drop these two files in the /newspack-plugin/languages directory, change your site's language to German, and confirm some of the front-end RAS strings -- like the Sign In button, and Log Out link on My Account -- have 'translated' appended to them.

[fake-german-translation.zip](https://github.com/Automattic/newspack-plugin/files/12621781/fake-german-translation.zip)

### Other information:
